ddev/src/ddev/repo/core.py

```diff
@@ -18,3 +27,38 @@ def name(self) -> str:
     @property
     def path(self) -> Path:
         return self.__path
+
+    @property
+    def integrations(self) -> IntegrationRegistry:
+        return self.__integrations
+
+    @cached_property
+    def config(self) -> RepositoryConfig:
+        from ddev.repo.config import RepositoryConfig
+
+        config_file = self.path / CONFIG_DIRECTORY / 'config.toml'
+        if not config_file.is_file():
+            return RepositoryConfig({})
+
+        from ddev.utils.toml import load_toml_file
+
+        return RepositoryConfig(load_toml_file(config_file))
```

Comment:
Suggestion:  Add an alternative constructor to `RepositoryConfig`, so that we can i.e.
```suggestion
        return RepositoryConfig.from_toml_file(config_file)
```

Reply:
Addressed!